frontend/src/lib/FileHelper.test.ts

```diff
@@ -15,16 +15,29 @@
  * limitations under the License.
```

Comment:
Is the test run only one platform?

Reply:
Our CI runs on ubuntu and I ran it on Mac but what really matters here is probably to test it on a Windows which we haven't done :(

Reply:
Oh, I see. So, you found the discrepancy this way.